saleor/graphql/core/types/common.py

```diff
@@ -435,7 +437,10 @@ class File(graphene.ObjectType):
 
     @staticmethod
     def resolve_url(root, info):
-        return info.context.build_absolute_uri(urljoin(settings.MEDIA_URL, root.url))
+        # check if URL is absolute:
+        if urlparse(root.url).netloc:
```

Comment:
Do we anticipate a situation, that `File` or `Image` object will have absolute URL? (I recall discussion with @patrys that we will have `UrlField` at some point, but that will not touch `File` and `Image` at all?)

Reply:
For swatch attributes, we'll have. And we don't want to change anything for swatches.